adapters/handlers/grpc/v1/prepare_reply.go

```diff
@@ -29,7 +29,7 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
-func searchResultsToProto(res []interface{}, start time.Time, searchParams dto.GetParams, scheme schema.Schema) (*pb.SearchReply, error) {
+func searchResultsToProto(res []interface{}, start time.Time, searchParams dto.GetParams, scheme schema.Schema, UsesPropertiesMessage bool) (*pb.SearchReply, error) {
```

Comment:
can we change the argument variable name to start with lowercase letter?

Reply:
Absolutely, sorry for the rookie syntax mistake 😅